src/main/webapp/i18n/de/featureOverview.json

```diff
@@ -3,18 +3,18 @@
         "title": "Onlineklausuren in Artemis",
         "featureDetails": "Featuredetails",
         "instructor": {
-            "pageTitle": "Klausurfeatures für Instruktoren",
-            "featureOverview": "Features für Instruktoren",
+            "pageTitle": "Klausurfeatures für Lehrende",
+            "featureOverview": "Features für Lehrende",
```

Comment:
From now in, I don't mark "Lehrende" or "Lehrperson" anymore :) .. I think these terms would be findable :)